backend/bitboxbase/bitboxbase.go

```diff
@@ -185,15 +220,15 @@ func (base *BitBoxBase) ConnectElectrum() error {
 
 // Deregister calls the backend's BitBoxBaseDeregister callback and sends a notification to the frontend, if bitboxbase is active.
 // If bitboxbase is not active, an error is returned.
-func (base *BitBoxBase) Deregister() (bool, error) {
-	if base.active {
-		// let the frontend know that the base is disconnected
-		base.fireEvent("disconnect")
-		base.onUnregister(base.bitboxBaseID)
-		base.active = false
-		return true, nil
-	}
-	return false, errp.New("Attempted call to non-active base")
+func (base *BitBoxBase) Deregister() error {
+	if !base.active {
+		return errp.New("Attempted call to non-active base")
+	}
+	// let the frontend know that the base is disconnected
+	base.fireEvent("disconnect")
```

Comment:
`"disconnect"` is a magic string, but should be a `bitboxbasestatus.Event`. Please move it to a definition with docs in bitboxbase/status.go.

I wish Go would not automatically cast between type aliases :)

Reply:
fyi: I just refactored the `if !base.active {}` part of `Deregister()`. This was in the code base already. 

But can do.